File: Beginner_Projects/Patterns/Alphabats/Python/G.py
for row in range(5):
    for col in range(5):
        if(row == 4 and (col != 0 and col != 3)) or (row*col == 0) or (row == 3 and col > 2):
            print('*', end=" ")
        print(end=" ")
    print()
input()

File: Beginner_Projects/Patterns/Alphabats/C++/G.cpp
#include <conio.h>
#include <iostream>
using namespace std;

int main()
{
    int row = 0;
    while (row < 5)
    {
        int col = 0;
        while (col < 6)
        {
            if ((row * col == 0) || (row == 4 && (col != 0 && col != 4)) ||
                (row == 3 && col > 1))
            {
                cout << "*";
            }
            else
            {
                cout << " ";
            }
            col++;
        }
        cout << endl;
        row++;
    }
    getch();
}

// Created by Salfi Hacker Mansoor Bukhari